justfile

```diff
@@ -25,37 +28,41 @@ venv:
 
 # Download data from RNAcentral
 download:
-    curl -O https://ftp.ebi.ac.uk/pub/databases/RNAcentral/r2dt/1.4/cms.tar.gz
+    curl -O https://ftp.ebi.ac.uk/pub/databases/RNAcentral/r2dt/{{ data_version }}/cms.tar.gz
     tar -xzf cms.tar.gz
 
 # Run shell in docker
 run:
-  docker run {{platform}} -v $(pwd):/rna/r2dt -v {{data_dir}}:/rna/r2dt/data/cms -it --rm {{image}}
+    docker run {{ platform_arg }} -v $(pwd):/rna/r2dt -v {{ data_dir }}:/rna/r2dt/data/cms -it --rm {{ image }}
 
 # Run all tests in docker
 test-all:
-  docker run {{platform}} --rm -it -v ./:/rna/r2dt/ -v {{data_dir}}:/rna/r2dt/data/cms {{image}} bash -c "./r2dt.py test"
+    docker run {{ platform_arg }} --rm -it -v ./:/rna/r2dt/ -v {{ data_dir }}:/rna/r2dt/data/cms {{ image }} bash -c "./r2dt.py test"
+
 # Run specific test in docker
 test TEST:
-  docker run {{platform}} --rm -it -v ./:/rna/r2dt/ -v {{data_dir}}:/rna/r2dt/data/cms {{image}} bash -c "./r2dt.py test Test{{TEST}}"
+    docker run {{ platform_arg }} --rm -it -v ./:/rna/r2dt/ -v {{ data_dir }}:/rna/r2dt/data/cms {{ image }} bash -c "./r2dt.py test Test{{ TEST }}"
 
 # Build R2DT Docker image
-build:
-  docker buildx build {{platform}} -t {{image}}  .
+build *BUILD_ARGS:
+    docker buildx build {{ platform_arg }} {{ BUILD_ARGS }} -t {{ image }}  .
 
-# Build base image
+# Build base image locally
 bbuild:
-  docker buildx build {{platform}} -t {{base_image}} base_image
+    docker buildx build {{ platform_arg }} -t {{ base_image }} base_image
+
+# Build base and then the r2dt images locally
+full-build: bbuild (build "--build-arg BASE_IMAGE_VERSION=latest")
```

Comment:
Is this a good place to use the `tag` variable?

Reply:
Not really, as explained above